### PR TITLE
Updated layout and ranking

### PR DESCRIPTION
**Pull request by timgdavies**
On this branch I've implemented:

* Updated weightings;
* Updated layout and display;
* New front page;
* Some debug information displayed when run locally

I would like to get this deployed by Thursday, ideally without the alpha banner.

### CODE REVIEW

File: prefix_finder/frontend/templates/home.html [hunk 1]
```diff
@@ -7,53 +7,53 @@
...
             </select>
-            {% if lookups.substructure %}
+            {% if lookups.subnational %}
               <div class="indented-option">
-                <div class="select-label">Organisation subtype</div>
-                <select name="substructure" class="selectpicker" data-live-search="true" data-width="280px" onchange="this.form.submit()">
-                <option value="">Any subtype</option>
-                {% for code, title, disabled in lookups.substructure %}
-                  <option {% if query.substructure == code %} selected {% elif disabled %} disabled {% endif %} value="{{code}}">{{title}}</option>
+                <div class="select-label">In the state of...</div>
```
Comment (edugomez) [inline prefix_finder/frontend/templates/home.html]: I find this label a bit odd in some case, i.e. for UK you are prompted to choose a state

Comment (timgdavies) [inline prefix_finder/frontend/templates/home.html]: Good point. Will edit to 'state or region of'

File: prefix_finder/frontend/templates/results.html [hunk 2]
```diff
@@ -1,31 +1,63 @@
 {% load static %}
 {% load results %}
 
+<div class="row">
 {% for result in results %}
+  <div class="col-xs-12 col-md-6">
   <div class="panel panel-default">
-    <div class="panel-heading">
-      <h4><a href="{% url 'list' result.code %}" data-toggle="tooltip" data-placement="top" title="More info">{{result.name.en}} ({{result.code}})</a></h4>
+    <div class="panel-heading"><span class="badge pull-right" data-toggle="popover" data-placement="bottom" title="Quality score" data-html="true" data-content="{%for key,value in result.quality_explained.items %}{{ key }} - {{ value }}<br/>{%endfor%}" data-trigger="hover">Quality score: {{ result.quality }}</span>
```
Comment (edugomez) [inline prefix_finder/frontend/templates/results.html]: Some missing spaces between `{%` and the following keyword (eg. `{%for key ...`)

File: prefix_finder/frontend/templatetags/results.py [hunk 3]
```diff
@@ -75,3 +77,52 @@ def tidy_results(results, length=None):
...
+        value = [str(item) for item in value]
+
+        if len(value) == 1:
+            return value[0]
+
+        # join all but the last element
+        all_but_last = ", ".join(value[:-1])
+        return "%s and %s" % (all_but_last, value[-1])
+    else:
+        return ""
```
Comment (edugomez) [inline prefix_finder/frontend/templatetags/results.py]: I think it is nice to have a filter for this, but the two filters are very similar and probably can be collapsed into one. You are only using the `lower=True` once I think (line 23 , _results.html_ above), so one thing I would do is to get rid of the argument `lower` at `join_or` and instead pass an argument with 'and' or 'or' (maybe changing the name of the function to `join_by` or something like that). As for the lower, you can use Django's `lower` filter in the template itself (after eg `join_by:"or"` ), as I think Django only allows to pass one argument to filters.

File: prefix_finder/frontend/views.py [hunk 4]
```diff
@@ -14,11 +14,11 @@
...
 RELEVANCE = {
     "MATCH_DROPDOWN": 10,
-    "MATCH_DROPDOWN_ONLY_VALUE": 1,
+    "MATCH_DROPDOWN_ONLY_VALUE": 10,
     "MATCH_EMPTY": 2,
-    "RECOMENDED_THRESHOLD": 5,
-    "SUGGESTED_THRESHOLD": 5,
-    "SUGGESTED_QUALITY_THRESHOLD": 60
+    "RECOMENDED_RELEVANCE_THRESHOLD": 5,
+    "SUGGESTED_RELEVANCE_THRESHOLD": 35,
```
Comment (edugomez) [inline prefix_finder/frontend/views.py]: Missing an _M_ in RECOMMENDED (nitpicking!)

File: prefix_finder/frontend/templatetags/results.py (hunk 3 above)
Comment (timgdavies) [inline prefix_finder/frontend/templatetags/results.py]: Ok. I might have to leave this for later - as not sure I've got time to refactor before Thursday.

**Comment by timgdavies** [comment]
I've fixed most of the issues above. I've not merged yet, as not sure if that should only happen when @Bjwebb is ready to deploy.

File: prefix_finder/frontend/templatetags/results.py (hunk 3 above)
Comment (edugomez) [inline prefix_finder/frontend/templatetags/results.py]: I can do this quickly and commit on your behalf, but perhaps you prefer to come back to this yourself at a later date?

Comment (timgdavies) [inline prefix_finder/frontend/templatetags/results.py]: If you can do this quickly that would be fantastic.

Comment (edugomez) [inline prefix_finder/frontend/templatetags/results.py]: I will

**Pull request merged**